File: legacy/src/main/java/br/com/legacy/managers/dtos/SearchFilterInput.java
package br.com.legacy.managers.dtos;

import java.io.Serializable;

public class SearchFilterInput implements Serializable {

    public String categoryId;
    public String subCategoriesId;
    public Long fromDate;
    public Long toDate;
    public String state;
    public String city;
    public String lowestPrice;
    public String highestPrice;

    public String to() {
        return categoryId + subCategoriesId + fromDate + toDate + state + city + lowestPrice + highestPrice;
    }
}

File: component_model_club/src/main/java/br/com/goin/component_model_club/model/SearchFilterInput.kt
package br.com.goin.component_model_club.model

import java.io.Serializable

class SearchFilterInput : Serializable {

    var categoryId: String? = null
    var subCategoriesId: String? = null
    var fromDate: Long? = null
    var toDate: Long? = null
    var state: String? = null
    var city: String? = null
    var lowestPrice: String? = null
    var highestPrice: String? = null

    fun to(): String {
        return categoryId + subCategoriesId + fromDate + toDate + state + city + lowestPrice + highestPrice
    }
}
